### PR TITLE
Make legs expandable

### LINKED ISSUE #487: Make Legs expandable
Acceptance criteria:
- arrows on right
- additional info is shown upon expanding the leg

- IMPORTANT: we are only adding the expanding fuctionality, the complete additional info will be covered in https://github.com/kiwicom/smart-faq/issues/513

screen shot 2018-06-25 at 16 54 37

When expanded:

screen shot 2018-06-27 at 17 21 57

Storybook on FE: https://bit.ly/2Kjxmcy

### PR DESCRIPTION
**Pull request by vborszeki**
closes #487<br/><br/><br/><url>LiveURL: https://smartfaq-487-expandable-legs.surge.sh</url>

### CODE REVIEW

**Comment by kantord** [comment]
rebase pls

File: src/SingleBookingPage/bookingItem/AccordionLegCities.js [hunk 1]
```diff
@@ -19,6 +20,10 @@ const citiesStyle = css`
     margin-bottom: 15px;
     align-items: space-between;
     padding: 7px 13px;
+    cursor: pointer;
+  }
+  div.legCities:focus {
+    outline: none;
```
[the remark this answers is not in the document]
Comment (vborszeki) [reply src/SingleBookingPage/bookingItem/AccordionLegCities.js]: Agree with making the accordions more accessible.

[the remark this answers is not in the document]
Comment (kantord) [reply src/SingleBookingPage/bookingItem/AccordionLegCities.js]: hmm it seems that I can't even "click" the cards with a keyboard tho, or expand the leg list for that matter. Focusing 'Log out' logs me out. Looks like our site is completely broken for keyboard users.

[the remark this answers is not in the document]
Comment (kantord) [reply src/SingleBookingPage/bookingItem/AccordionLegCities.js]: So maybe we should create a bunch of issues about this?

[the remark this answers is not in the document]
Comment (vborszeki) [reply src/SingleBookingPage/bookingItem/AccordionLegCities.js]: Yes, I was thinking about the same, to create another issue fixing accessibility. And leaving the focus outline for keyboard users could also be solved there.

**Pull request merged**